### PR TITLE
[vcpkg baseline][paraview:x64-osx][vcpkg_cmake_config_fixup] Revert #19469

### PR DESCRIPTION
**Pull request by PhoebeHui**
Revert OSX related changes in vcpkg_cmake_config_fixup and vcpkg-cmake-config in https://github.com/microsoft/vcpkg/pull/19469.

This PR expects to resolve the CI baseline regression 'paraview:x64-osx', since it blocks many PRs, we'd better to revert it now.

Confrimed with Jack, the changes will be re-test and fixed via PR https://github.com/microsoft/vcpkg/pull/19629.

### CODE REVIEW

File: versions/baseline.json [hunk 1]
```diff
@@ -6618,7 +6618,7 @@
     },
     "vcpkg-cmake-config": {
       "baseline": "2021-08-11",
-      "port-version": 0
+      "port-version": 1
```
Comment (Neumann-A) [inline versions/baseline.json:6621]: Just edit the date instead?

File: ports/vcpkg-cmake-config/vcpkg.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {
   "name": "vcpkg-cmake-config",
-  "version-date": "2021-08-11"
+  "version-date": "2021-08-11",
+  "port-version": 1
```
Comment (Neumann-A) [inline ports/vcpkg-cmake-config/vcpkg.json:4]: also edit the date instead of pushing the port-version

File: versions/baseline.json (hunk 1 above)
Comment (PhoebeHui) [reply versions/baseline.json]: Updated! thanks for your review!

File: ports/vcpkg-cmake-config/vcpkg.json (hunk 2 above)
Comment (PhoebeHui) [reply ports/vcpkg-cmake-config/vcpkg.json]: Updated! thanks for your review!

**Review by JackBoosY — APPROVED** [review]

**Comment by JackBoosY** [comment]
Ping @strega-nil-ms for merge this first.

**Comment by PhoebeHui** [comment]
@dan-shaw @strega-nil, could you help merge this PR?

**Comment by PhoebeHui** [comment]
Thank you Nicole! ☺

File: versions/baseline.json [hunk 3]
```diff
@@ -6617,8 +6617,8 @@
       "port-version": 0
     },
     "vcpkg-cmake-config": {
-      "baseline": "2021-08-11",
-      "port-version": 0
+      "baseline": "2021-05-22",
+      "port-version": 1
```
Comment (JackBoosY) [inline versions/baseline.json:6621]: @strega-nil-ms We couldn't do that because the version shoud only be added, not be reverted.

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: versions/baseline.json (hunk 3 above)
Comment (strega-nil) [reply versions/baseline.json:6621]: I think that it's fine, but would like confirmation from @ras0219-msft

Comment (dg0yt) [reply versions/baseline.json:6621]: It may look fine now, but with the next change, we will have two variants of port version 2...

Comment (dg0yt) [reply versions/baseline.json:6621]: Cut the "port version 2", it is different "version-date". It is really easy to start confusion ;-)

**Comment by PhoebeHui** [comment]
@strega-nil, the CI pipeline failed with following failures:
```
versions/v-/vcpkg-cmake-config.json(2,): error : While reading versions for port vcpkg-cmake-config from file: version/v-/vcpkg-cmake-config.json
Local port version `2021-05-22#1` exists in version file but it's not the first entry in the "versions" array.
```

**Comment by JackBoosY** [comment]
I think we can drop this PR and take https://github.com/microsoft/vcpkg/pull/19629 instead.

**Pull request merged**